### DISCUSSION TITLE
Great job, everyone!

### DISCUSSION THREAD

**Discussion by EchterAgo** [openzfsonwindows/openzfs#320] [Testing]
For the first time since trying this back in August I feel good about actually storing my backups on it.

I still have performance problems with the latest code, but I haven't found any issue regarding data integrity or being unable to mount. 

Thanks to @lundman and @andrewc12, I learned a lot from both of you.

**Comment by lundman** [comment]
Big thanks to you for sticking with it, helping us find bugs, filing reports and bug-producers.
